tests/test_logger_config.py

```
@@ -120,3 +120,92 @@ def test_setup_file_handler_adds_handler(self, tmp_path):
+        mock_log_dir = MagicMock()
+        mock_path_instance.__truediv__ = MagicMock(return_value=mock_log_dir)
+
+        configure_logging(False)
+        # Check that setup_logging was called with INFO level
+        call_args = mock_setup_logging.call_args
+        assert call_args[1]["level"] == logging.INFO
+
+    @patch("src.logger_config.setup_logging")
+    @patch("src.logger_config.Path")
+    @patch.dict("os.environ", {}, clear=False)
+    def test_configure_logging_uses_default_main_dir(self, mock_path_class, mock_setup_logging):
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_setup_logging`

(ARG002)